darts/tests/models/forecasting/test_exponential_smoothing.py

```diff
@@ -91,3 +91,31 @@ def test_fit_kwargs(self):
         # forecasts should be slightly different
         assert pred.time_index.equals(pred_ls.time_index)
         assert all(np.not_equal(pred.values(), pred_ls.values()))
+
+    def test_random_errors(self):
+        """Test whether random_errors parameter is correctly passed to simulate()"""
+        series = tg.sine_timeseries(length=100, freq="H")
+        model = ExponentialSmoothing()
```

Comment:
For safety, let's include random_state to all four model creations as well to make sure that the different predictions do not come from randomness during training.
```suggestion
        model = ExponentialSmoothing(random_state=42)
```

and in the occurrences below